keepassxc-browser/content/keepassxc-browser.js

```diff
@@ -1080,19 +1080,28 @@ kpxc.preparePageForMultipleCredentials = function(credentials) {
         return;
     }
 
-    function getLoginText(credential) {
+    function getLoginText(credential, withGroup) {
+        const group = (withGroup && credential.group) ? `[${credential.group}] ` : '';
         const visibleLogin = (credential.login.length > 0) ? credential.login : tr('credentialsNoUsername');
+        const text = `${group}${credential.name} (${visibleLogin})`;
```

Comment:
This reverses the order of the credential display... should maintain that order: 
`${visibleLogin} (${group}${credential.name})`